src/generators/clientContextFileGenerator.ts

```diff
@@ -246,5 +246,8 @@ function getRequiredParamChecks(requiredParameters: ParameterDetails[]) {
 }
 
 function getRequiredParamAssignments(requiredParameters: ParameterDetails[]) {
-  return requiredParameters.map(({ name }) => `this.${name} = ${name};`);
+  const blacklistClientParameters = ["credentials"];
```

Comment:
Hate to be the word police, but could we use a different name like "disallowedClientParameters"?

Reply:
Updated!